lib/iris/analysis/_regrid.py

```diff
@@ -545,3 +855,121 @@ def __call__(self, src):
                                    sample_grid_x, sample_grid_y,
                                    regrid_callback)
         return result
+
+
+class CurvilinearRegridder(object):
```

Comment:
Maybe being a little fussy but do you mind shifting this so that it is above RectilinearRegridder. That way they are in alphabetical order